settings.gradle.kts

```diff
@@ -1 +1,9 @@
-rootProject.name = "sigstore-java"
+rootProject.name = "sigstore-java-root"
+
+includeBuild("build-logic-commons")
+includeBuild("build-logic")
+
+include("sigstore-java")
+include("sigstore-sign-base-plugin")
```

Comment:
should these projects have `gradle` in their name, if we're going to be including maven stuff in here too at some point? alternatively could all the gradle stuff go in a `gradle` subdir -> `sigstore-java/gradle`

Reply:
makes sense

Reply:
`gradle` folder alone won't help much since the project would have to be renamed anyway.

Let us have `gradle/sigstore-gradle-...` naming:

```kotlin
include("gradle:sigstore-gradle-sign-base-plugin")
include("gradle:sigstore-gradle-sign-plugin")
```

Reply:
I went with `sigstore-gradle:sigstore-gradle-sign-base-plugin`